src/tests/__snapshots__/rockets.test.js.snap

```diff
@@ -0,0 +1,70 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`testing snapshots of Rocket component testing Rockets component 1`] = `
+Object {
+  "asFragment": [Function],
+  "baseElement": <body>
+    <div>
+      <h1>
+        Loading...
+      </h1>
+    </div>
+  </body>,
+  "container": <div>
+    <h1>
+      Loading...
+    </h1>
+  </div>,
+  "debug": [Function],
+  "findAllByAltText": [Function],
+  "findAllByDisplayValue": [Function],
+  "findAllByLabelText": [Function],
+  "findAllByPlaceholderText": [Function],
+  "findAllByRole": [Function],
+  "findAllByTestId": [Function],
+  "findAllByText": [Function],
+  "findAllByTitle": [Function],
+  "findByAltText": [Function],
+  "findByDisplayValue": [Function],
```

Comment:
[OPTIONAL] I don't know how the previous code reviewer missed this error Your test seem to be having an issue please fix this error.

Take a look at the screen shot below:

![Screenshot 2023-03-09 at 8 31 51 PM](https://user-images.githubusercontent.com/79506105/224126644-94109d75-5246-4261-9ec8-b1c1b82fae34.png)

This has been marked optional because it wasn't a requested change fro the previous code reviewer but I strong emphasise you work on these changes even if they are marked as optional.

Reply:
Hi @KayLemba this is how test looks like on my side before any requested code update
![image](https://user-images.githubusercontent.com/24703171/224170553-a8a5e6c0-7a7c-44bd-bba3-49246eb7cfa6.png)

Reply:
This problem happens because there are code updates in the component so you need to regenerate the snapshot and every test will pass. This is how it looks on my side after regenerating the snapshot of the failed snapshot.
![image](https://user-images.githubusercontent.com/24703171/224171655-6d421e78-d96c-49e5-ad54-fe94b84641d9.png)

Reply:
I'm not quite sure what to fix here because the tests pass when I run it on my machine.